tools/downloader/list_topologies.yml

```diff
@@ -1570,6 +1570,31 @@ topologies:
     framework: "tf"
     license: https://raw.githubusercontent.com/tensorflow/models/master/LICENSE
 
+  - name: "caffenet"
+    description: >-
+      CaffeNet model. Used for classification. For details see paper
+      <https://arxiv.org/abs/1408.5093>.
+    task_type: "classification"
+    files:
+      - name: bvlc_reference_caffenet.caffemodel
+        sha256: 472d4a06035497b180636d8a82667129960371375bd10fcb6df5c6c7631f25e0
+        source: http://dl.caffe.berkeleyvision.org/bvlc_reference_caffenet.caffemodel
+      - name: deploy.prototxt
+        sha256: 922248a4d2f6aac1cc8e7e5dbd996cc2ecd3356480d67c198f6cf96b12311a04
+        source: https://raw.githubusercontent.com/BVLC/caffe/2cc3844cb2a4a72de10d321781dc8f994bef95c7/models/bvlc_reference_caffenet/deploy.prototxt
+    output: "classification/caffenet/cf/"
+    model_optimizer_args:
+      - --framework=caffe
+      - --data_type=FP32
+      - --input_shape=[1,3,227,227]
+      - --input=data
+      - --mean_values=data[104.0,117.0,123.0]
+      - --output=prob
+      - --input_model=$dl_dir/bvlc_reference_caffenet.caffemodel
+      - --input_proto=$dl_dir/deploy.prototxt
+    framework: "caffe"
+    license: https://raw.githubusercontent.com/BVLC/caffe/master/LICENSE
```

Comment:
This isn't the correct license. The license for the model itself is in https://raw.githubusercontent.com/BVLC/caffe/master/models/bvlc_reference_caffenet/readme.md. See `alexnet` for a similar example.